backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -43,52 +46,51 @@
-        "<b>Workflow description: </b>" + sanitize(workflow),
-        "<b>Does the person who checks people in also record test results: </b>"
-            + sanitize(recordsTestResults),
-        "<b>Single person registration and testing process time: </b>" + sanitize(processTime),
-        "<b>Time spent submitting results daily: </b>" + sanitize(submittingResultsTime),
-        "",
-        "<h1>Ordering provider</h1>",
-        "<b>First name: </b>" + sanitize(opFirstName),
-        "<b>Last name: </b>" + sanitize(opLastName),
-        "<b>National Provider Identifier (NPI): </b>" + sanitize(npi),
-        "<b>Phone number: </b>" + sanitize(opPhoneNumber),
-        "<b>Street address: </b>" + sanitize(opMailingAddress1),
-        "<b>Unit type: </b>" + sanitize(opAptSuiteOther),
-        "<b>Unit number: </b>" + sanitize(opAptFloorSuiteNo),
-        "<b>City: </b>" + sanitize(opCity),
-        "<b>State: </b>" + sanitize(opState),
-        "<b>Zip Code: </b>" + sanitize(opZip),
-        "<b>County: </b>" + sanitize(opCounty));
+  @Override
+  public String getTemplateName() {
+    return TEMPLATE_NAME;
+  }
+
+  @Override
+  public Map<String, Object> toTemplateVariables() {
+    Map<String, Object> variableMap = new HashMap<>();
+
+    variableMap.put("firstName", firstName);
+    variableMap.put("lastName", lastName);
+    variableMap.put("email", email);
+    variableMap.put("workPhoneNumber", workPhoneNumber);
+    variableMap.put("cellPhoneNumber", cellPhoneNumber);
+    variableMap.put("mailingAddress1", mailingAddress1);
+    variableMap.put("aptSuiteOther", aptSuiteOther);
+    variableMap.put("aptFloorSuiteNo", aptFloorSuiteNo);
+    variableMap.put("city", city);
+    variableMap.put("state", state);
+    variableMap.put("zip", zip);
+    variableMap.put("county", county);
+    variableMap.put("facilityType", facilityType);
+    variableMap.put("organizationName", organizationName);
+    variableMap.put("facilityName", facilityName);
+    variableMap.put("cliaNumber", cliaNumber);
+    variableMap.put("testingDevices", testingDevices);
+    variableMap.put("accessDevices", accessDevices);
+    variableMap.put("browsers", browsers);
+    variableMap.put("workflow", workflow);
+    variableMap.put("recordsTestResults", recordsTestResults);
+    variableMap.put("processTime", processTime);
+    variableMap.put("submittingResultsTime", submittingResultsTime);
+    variableMap.put("opFirstName", opFirstName);
+    variableMap.put("opLastName", opLastName);
+    variableMap.put("npi", npi);
+    variableMap.put("opPhoneNumber", opPhoneNumber);
+    variableMap.put("opMailingAddress1", opMailingAddress1);
+    variableMap.put("opAptSuiteOther", opAptSuiteOther);
+    variableMap.put("opAptFloorSuiteNo", opAptFloorSuiteNo);
+    variableMap.put("opCity", opCity);
+    variableMap.put("opState", opState);
+    variableMap.put("opZip", opZip);
+    variableMap.put("opCounty", opCounty);
```

Comment:
Hmm... could we use [Jackson's Object mapper](https://mkyong.com/java/java-convert-object-to-map-example/) here to make things simpler?
```
ObjectMapper mapObject = new ObjectMapper();
return mapObject.convertValue(this, Map.class);
```

Reply:
Oh, yeah, that is nice!   Unfortunately, here it seems that using ObjectMapper would force our property names to be snake case because of the `JsonNaming` annotation.  After looking into it a bit it seems like ObjectMapper will respect the annotation and the naming cannot be overridden by `setPropertyNamingStrategy`.

Reflection works nicely for producing the map without much code, but maybe we should wait and see if the templates get used more first.  If we need it more, we could add a default implementation using reflection (but sonarcloud won't like it!).

Reply:
Added https://github.com/CDCgov/prime-simplereport/issues/1282 to track this improvement.